### PR TITLE
Aurora config

### PR DESCRIPTION
**Pull request by nbwhite**

### CODE REVIEW

**Review by sindhu-106 — APPROVED** [review]

File: configurations/aurora/SystemManifest.json [hunk 1]
```diff
@@ -4355,7 +4355,92 @@
...
+              "definition": "olympus_ecb",
+              "name": "CN5",
+              "hostname": "s50b0n0",
+              "x": 0,
+              "y": 6
+            },
+            {
+              "definition": "olympus_ecb",
+              "name": "CN4",
+              "hostname": "s40b0n0",
```
Comment (yhuan47) [inline configurations/aurora/SystemManifest.json:4411]: s40b0n0 or s4b0n0?

File: configurations/aurora/scripts/Aurora.yml [hunk 2]
```diff
@@ -410,7 +410,7 @@ views:
           x: 0
           y: 0
 
-# Compute Node Racks Row 8
+# Compute Node Racks Row 1
         - definition: olympus_rack
           name: X[4000-4002]
           xname: x[4000-4002]
```
Comment (yhuan47) [inline configurations/aurora/scripts/Aurora.yml:416]: CDU406 then CDU107, is it supposed to be CDU407? please double check. Thanks!

File: configurations/aurora/SystemManifest.json (hunk 1 above)
Comment (nbwhite) [reply configurations/aurora/SystemManifest.json]: Good catch, fixed.

File: configurations/aurora/scripts/Aurora.yml (hunk 2 above)
Comment (nbwhite) [reply configurations/aurora/scripts/Aurora.yml:416]: I checked the cdu, and it had no label, and doesn't show up in any way in the hosts. It looks like the cdu in question was labelled that way in an older diagram.

**Review by sindhu-106 — APPROVED** [review]

**Review by yhuan47 — APPROVED** [review]

**Pull request merged**